sklearn/model_selection/tests/test_validation.py

```diff
@@ -253,6 +253,7 @@ def fit(
                 P.shape[0],
                 P.shape[1],
             )
+        self.fitted_ = True
```

Comment:
Why do we need it here?

Reply:
cause `check_is_fitted` checks for an attribute with a trailing underscore. This makes `check_is_fitted(self)` to pass.